src/Docker/Tests/Manager/ImageManagerTest.php

```diff
@@ -56,7 +56,7 @@ public function testDelete()
     {
         $manager = $this->getManager();
 
-        $image = $manager->find('ubuntu', 'precise');
+        $image = $manager->find('ubuntu', 'vivid');
```

Comment:
I think deleting didn’t work in the test because of other images depending on `ubuntu:precise`. This fixes the errors.